src/cljdoc/render/articles.clj

```diff
@@ -33,12 +33,17 @@
    (when (seq doc-bundle)
      (->> doc-bundle
           (map (fn [doc-page]
-                 (let [slug-path (-> doc-page :attrs :slug-path)]
+                 (let [slug-path (-> doc-page :attrs :slug-path)
+                       external-url (-> doc-page :attrs :cljdoc.doc/external-url)]
                    [:li
                     {:class (when (seq (:children doc-page)) "mv2")}
+                    (when external-url
+                      [:img.v-mid.mr1 {:src "https://microicon-clone.vercel.app/external/12"}])
```

Comment:
I'm not sure what this looks like but I'm guessing this puts the image to the left of the link. I think we'd want it on the right of the link, though? I guess I'd have to see it. I can futz with the styling later if you don't want to, really.